skglm/experimental/pdcd_ws.py

```diff
@@ -0,0 +1,242 @@
+import warnings
+
+import numpy as np
+from numpy.linalg import norm
+from scipy.sparse import issparse
+
+from numba import njit
+from skglm.utils.jit_compilation import compiled_clone
+from sklearn.exceptions import ConvergenceWarning
+
+
+class PDCD_WS:
```

Comment:
you need to merge upstream/main into your PR branch for these to disappear from the diff

Reply:
Yes indeed!